### ISSUE TITLE
Rule to enforce a test has tags

### ISSUE THREAD

**Issue by seanlafferty-ibm** [playwright-community/eslint-plugin-playwright#401]
We currently use tags as a way to control what environment a test gets ran in, so we require all tests have tags. However, I don't believe this is enforceable by a lint rule.

I can config the tag rule like:
```ts
"playwright/valid-test-tags": [
  "error",
  { allowedTags: ["@saas", "@onprem"] },
],
```
but that still permits a test like `test("Im a test with no tags")`. Any thoughts about this? Perhaps a `allowTestsWithoutTags` bool option for this rule?

I suppose there could be some nuance in that a `test()` inherits the tags of its parent `describe()` block, so if this rule did exist, we'd have to decide if we want do the recursion to make
```ts
describe("parent @tag1")
  test("child")
```
permissable, or just have the rule operate naively on `test()` calls.
